packages/next/src/server/dev/turbopack/manifest-loader.ts

```diff
@@ -268,27 +268,33 @@ export class TurbopackManifestLoader {
     )
   }
 
-  private mergeBuildManifests(manifests: Iterable<BuildManifest>) {
-    const manifest: Partial<BuildManifest> & Pick<BuildManifest, 'pages'> = {
-      pages: {
-        '/_app': [],
-      },
+  private mergeBuildManifests(
+    manifests: Iterable<BuildManifest>
+  ): Partial<BuildManifest> & Pick<BuildManifest, 'pages'> {
+    const polyfillFiles: string[] = []
+    const pages = {
+      '/_app': [],
+    }
+    let rootMainFiles: readonly string[] = []
+    for (const m of manifests) {
+      Object.assign(pages, m.pages)
+      if (m.rootMainFiles.length) rootMainFiles = m.rootMainFiles
+      polyfillFiles.push(...m.polyfillFiles)
```

Comment:
Probably want to avoid duplicates here similar to rootMainFiles

Reply:
Ah, okay, deduping makes a lot of sense. With the current code, I do end up with duplicates in `polyfillFiles` after I try loading multiple pages.

Truth be told, I wasn't quite sure what to do here because

```
if (m.rootMainFiles.length) rootMainFiles = m.rootMainFiles
```

doesn't seem quite right to me either unless we know for a fact that `rootMainFiles` is always the same value. Maybe we do, and so that's fine, but it's not clear that it's intentional.

We can probably know that's true for `polyfillFiles`, but it still seem a little ugly not to merge these fields properly.